core/src/main/java/com/xmartlabs/bigbang/core/helper/gsonadapters/LocalDateCustomFormatterAdapter.kt

```diff
@@ -0,0 +1,31 @@
+package com.xmartlabs.bigbang.core.helper.gsonadapters
+
+import com.google.gson.*
+import com.xmartlabs.bigbang.core.extensions.ifException
+import com.xmartlabs.bigbang.core.extensions.localDatefromEpochMilli
+import com.xmartlabs.bigbang.core.extensions.toInstant
+import org.threeten.bp.LocalDate
+import org.threeten.bp.format.DateTimeFormatter
+import timber.log.Timber
+import java.lang.reflect.Type
+
+/** [Gson] type adapter that serializes [LocalDate] objects to any specified format.  */
+class LocalDateCustomFormatterAdapter(private val dateFormat: DateTimeFormatter) :
+    JsonSerializer<LocalDate>, JsonDeserializer<LocalDate> {
+  @Synchronized
```

Comment:
cool, so the meaning is different than the lombok annotation meaning. it's the same of ` syncronized void someting()` java code, right?